R/generate_pt_nowcast_mat_list.R

```diff
@@ -1,19 +1,18 @@
 #' Generate retrospective nowcasts
 #'
 #' This function ingests a list of incomplete reporting triangles and
-#'   generates a list of reporting squares, or "complete"
-#'   point estimates of reporting triangles based on the delay estimated in
-#'   each triangle. It uses the specified `n` number of
-#'   observations to estimate the empirical delay for each retrospective
+#'   generates a list of point nowcast matrices, based on the delay estimated in
+#'   each triangle or the corresponding delay passed in. It uses the specified
+#'   `n` number of reference times to estimate the delay in each retrospective
 #'   reporting triangle.
 #'
 #' @param reporting_triangle_list List of reporting triangle matrices, in order
```

Comment:
Flagging another instance of inconsistent doces between this and generate_triangles. One is very verbose and one is not. They could be shared. Related to #144